cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -321,7 +322,21 @@ public TwillController start(long timeout, TimeUnit timeoutUnit) {
     }
   }
 
-  private V1Deployment createDeployment(V1ConfigMap configMap, long startTimeoutMillis) throws ApiException {
+  /**
+   * Return a {@link V1Deployment} object that will be deployed in Kubernetes to run the program
+   */
+  @VisibleForTesting
+  public V1Deployment buildDeployment(RunId runid,
```

Comment:
Usually method exposed for testing should be kept as package private instead of public.

Reply:
Fixed